wp-cli/vip-users.php

```diff
@@ -0,0 +1,155 @@
+<?php
+class VIP_Go_Users_Command extends WPCOM_VIP_CLI_Command {
+	/**
+	 * Changes the WordPress username (user_login)
+	 *
+	 * A lot of this functionality is borrowed or inspired from https://wordpress.org/plugins/username-changer/
+	 *
+	 * ## OPTIONS
+	 *
+	 * <username>
+	 * : The existing username of the user to rename.
+	 *
+	 * <newname>
+	 * : The new username to use.
+	 *
+	 * [--dry-run=<true>]
+	 * : Do a dry run by default, change to false to make permanent changes.
+	 *
+	 * 	 * eg.: `wp vip-go-users change-username jsmith johns`
+	 *
+	 * @subcommand change-username
+	 */
+	public function change_username( $args, $assoc_args ) {
+		// If --dry-run is not set, then it will default to true
+		// Must set --dry-run explicitly to false to run this command
+		if ( isset( $assoc_args['dry-run'] ) ) {
+			$dry_run = 'false' === $assoc_args['dry-run'] ? false : true;
+		} else {
+			$dry_run = true;
+		}
+
+		list( $old_username, $new_username ) = $args;
+
+		if ( username_exists( $new_username ) ) {
+			WP_CLI::error( 'New username already exists!' );
+		}
+
+		// One last sanity check to ensure the user exists
+		$user_id = username_exists( $old_username );
+		if ( $user_id ) {
+			if ( ! $dry_run ) {
+				global $wpdb;
+				// Update username!
+				$query_user_login = $wpdb->prepare( "UPDATE $wpdb->users SET user_login = %s WHERE user_login = %s", $new_username, $old_username );
+
+				if ( false !== $wpdb->query( $query_user_login ) ) {
+					// Update user_nicename
+					$query_user_nicename = $wpdb->prepare( "UPDATE $wpdb->users SET user_nicename = %s WHERE user_login = %s AND user_nicename = %s", $new_username, $new_username, $old_username );
+					$wpdb->query( $query_user_nicename );
+
+					// Update display_name
+					$query_display_name = $wpdb->prepare( "UPDATE $wpdb->users SET display_name = %s WHERE user_login = %s AND display_name = %s", $new_username, $new_username, $old_username );
```

Comment:
We should probably only change display name if it also matches `$old_username`, just like the nickname code does below.